manage.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'construct.settings')
+    try:
+        from django.core.management import execute_from_command_line
+    except ImportError as exc:

```

Comment:
where are the test cases??
where is aditya

Reply:
how are you